src/pacer/demo_callbacks.py

```
@@ -0,0 +1,111 @@
+            # Send notifications
+            
+        elif filing.filing_type.value == "order":
+            logger.info("🔨 COURT ORDER - Processing for compliance tracking")
+            # Update case status
+    
+    # Save to file for demo
+    try:
+        with open("detected_filings.json", "w") as f:
+            json.dump(DETECTED_FILINGS, f, indent=2)
+        logger.info(f"💾 Saved {len(DETECTED_FILINGS)} detected filings to detected_filings.json")
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)